app/model/OnlineTestCommands.scala

```diff
@@ -25,42 +25,46 @@ import reactivemongo.bson.{BSONDocument, BSONHandler, Macros}
 
 object OnlineTestCommands {
   case class Phase1Test(scheduleId: Int,
-                        usedForResults: Boolean,
-                        cubiksUserId: Int,
-                        testProvider: String = "cubiks",
-                        token: String,
-                        testUrl: String,
-                        invitationDate: DateTime,
-                        participantScheduleId: Int,
-                        startedDateTime: Option[DateTime] = None,
-                        completedDateTime: Option[DateTime] = None,
-                        resultsReadyToDownload: Boolean = false,
-                        reportId: Option[Int] = None,
-                        reportLinkURL: Option[String] = None,
-                        reportStatus: Option[String] = None)
+    usedForResults: Boolean,
+    cubiksUserId: Int,
+    testProvider: String = "cubiks",
+    token: String,
+    testUrl: String,
+    invitationDate: DateTime,
+    participantScheduleId: Int,
+    startedDateTime: Option[DateTime] = None,
+    completedDateTime: Option[DateTime] = None,
+    resultsReadyToDownload: Boolean = false,
+    reportId: Option[Int] = None,
+    reportLinkURL: Option[String] = None,
+    reportStatus: Option[String] = None,
+    testResult: Option[model.persisted.TestResult] = None
+  )
 
   object Phase1Test {
     import repositories.BSONDateTimeHandler
     implicit val phase1TestHandler: BSONHandler[BSONDocument, Phase1Test] = Macros.handler[Phase1Test]
     implicit val phase1TestFormat = Json.format[Phase1Test]
   }
 
-  case class Phase1TestProfile(
-                                expirationDate: DateTime,
-                                tests: List[Phase1Test]
-  ) {
-    def activeTests = tests filter (_.usedForResults)
+  trait TestProfile {
+    def expirationDate: DateTime
+    def tests: List[Phase1Test]
```

Comment:
Is this list meant to contain only Phase1Test? If so I don't see much benefit in using this trait but perhaps I'm missing a bit of background here.